src/webextension/background/views.js

```diff
@@ -91,6 +91,15 @@ export async function openView(name) {
 }
 
 export async function closeView(name) {
+  if (!name) {
+    console.log("closing all views");
+    Object.keys(views).forEach((v) => {
+      v = views[v];
+      v.close();
+    });
```

Comment:
Not sure how you feel about recursion, but this shaves a few chars (and keeps the close view logic in one less place):
```js
Object.keys(views).forEach(v => closeView(v));
```

Reply:
With the convention we're following here, it would be:

```javascript
Object.keys(views).forEach((v) => closeView(v));
```

But I don't have a strong opinion either way.